core/modules/contact/contact.admin.inc

```diff
@@ -24,33 +24,40 @@ function contact_settings_form($form, &$form_state) {
   );
   $form['fields']['copy'] = array(
     '#type' => 'checkbox',
-    '#title' => t("Show the 'Send yourself a copy' checkbox"),
-    '#description' => t("Allow users to send themselves a copy on site-wide and personal contact forms."),
+    '#title' => t("Allow users to send themselves a copy of the submitted data."),
+    '#description' => t("Exposes a 'Send yourself a copy' checkbox on site-wide and personal contact forms."),
     '#default_value' => contact_show_personal_copy_checkbox()
   );
 
   $form['threshold'] = array(
     '#type' => 'fieldset',
+    '#title' => t("Send limit"),
```

Comment:
Spam prevention

Reply:
I think we can use a more general heading here, and something that resonates more clearly with people.